### ISSUE TITLE
toggle menu aria-label

### ISSUE THREAD

**Issue by timkingbu** [bu-ist/responsive-foundation#276]
**Describe the bug**
“The aria-label should be used to provide a text alternative to an element that has no visible text on the screen.”

The button text can be customized in individual themes...
- [responsive_mega_menu_closed](https://github.com/search?q=org%3Abu-ist%20responsive_mega_menu_closed&type=code)
- [responsive_mega_menu_opened](https://github.com/search?q=org%3Abu-ist+responsive_mega_menu_opened&type=code)

...but [`toggle.js`](https://github.com/bu-ist/responsive-foundation/blob/5d5a11a7195dd129abe42c63ba187b41b12a24f9/burf-theme/js-dev/modules/toggle.js#L31-L35) that sets the `aria-label` does not share those hooks and always sets the label to Open menu or Close menu.

~Sloppy solution: Let the `js` control the label and use [aria-hidden](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden) to have screen readers ignore whatever text is displayed.~

**Optimal solution:** [Have the js pull the label from the inner text as described here](https://github.com/bu-ist/r3-law/issues/436#issuecomment-1572085271).

**To Reproduce**
Go to https://www.bu.edu/diversity/ and look at the code for the megamenu. 

<img width="1106" alt="Image" src="https://github.com/user-attachments/assets/2d5a7ab0-9918-4ceb-b564-a3d55cae0937" />

### See also

- https://github.com/bu-ist/responsive-framework-3x/issues/58
- https://github.com/bu-ist/r3-law/issues/436
- https://github.com/bu-ist/responsive-framework-3x/issues/59
- https://github.com/bu-ist/responsive-foundation/issues/137
